tests/transactional/install_updates.pm

```diff
@@ -47,14 +47,19 @@ sub run {
     }
 
     # First we update the system
+    record_info('Updates', script_output('zypper lu'));
     fully_patch_system(trup_call_timeout => 1800);
+    record_info('INFO', 'The system is patched...');
 
     # Now we add the incident repositories and do a zypper patch
     add_test_repositories;
+    record_info('INFO', "Test repositories added:\n" . script_output('zypper lr -u'));
     record_info('Updates', script_output('zypper lu'));
-    my $ret = trup_call('up', timeout => 300, proceed_on_failure => 1);
+    my $ret = trup_call('patch', timeout => 300, proceed_on_failure => 1);
```

Comment:
Why we have here `proceed_on_failure => 1`? I see it here for long time, but i don't understand why.

Reply:
Ok, it's because  of  https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17589  and https://bugzilla.suse.com/show_bug.cgi?id=1213991 is already fixed and workaround could be removed.